File: src/main/java/io/jsql/hazelcast/ReplicationCMD.java
package io.jsql.hazelcast;

import java.io.Serializable;

/**
 * Created by 长宏 on 2017/5/5 0005.
 */
public class ReplicationCMD implements Serializable{
    public long fromlsn;
    public long tolsn;
    public long id;

    public ReplicationCMD(long fromlsn, long tolsn, long id
    ) {
        this.id = id;
        this.fromlsn = fromlsn;
        this.tolsn = tolsn;
    }
}

File: src/main/java/io/jsql/hazelcast/ReplicationCMD.kt
/*
 * Java-based distributed database like Mysql
 */

package io.jsql.hazelcast

import java.io.Serializable

/**
 * Created by 长宏 on 2017/5/5 0005.
 */
class ReplicationCMD(var fromlsn: Long, var tolsn: Long, var id: Long
) : Serializable
